ui/src/apps/overview/RootComponent.js

```diff
@@ -40,9 +64,28 @@ const App = () => {
               <Card
                 size="small"
                 bordered={false}
-                title={t('overview.top_statements.title')}
+                title={
+                  <div style={{ display: 'flex', alignItems: 'baseline' }}>
```

Comment:
is it better to use `center`?

Reply:
Tried, but it looks like this:

![image](https://user-images.githubusercontent.com/1284531/78866811-a917ef80-7a72-11ea-9bfb-376fc3c16a5f.png)

because the h3 tag has some additional padding-bottom. (but of course, we can use div replace the h3 and increase its font-size).